src/lib/helpers/renderConfig.test.ts

```
@@ -0,0 +1,181 @@
+	component: TComponent;
+	props: ComponentProps<TComponent> | Record<string, never>;
+
+	constructor(
+		component: TComponent,
+		props: ComponentProps<TComponent> | Record<string, never> = {}
+	) {
+		this.component = component;
+		this.props = props;
+	}
+}
+
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)